packages/connection/src/Manager.php

```diff
@@ -1735,4 +1735,23 @@ public function sign_role( $role, $user_id = null ) {
 
 		return $role . ':' . hash_hmac( 'md5', "{$role}|{$user_id}", $token->secret );
 	}
+
+	/**
+	 * Some hosts disable the OpenSSL extension and so cannot make outgoing HTTPS requsets
+	 */
+	public static function fix_url_for_bad_hosts( $url ) {
```

Comment:
Nit: Could we add some php docs here? We try to keep the packaged code up to our highest standards.

Reply:
See a142b9c. I added a docblock with param and return.